README.markdown

```diff
@@ -30,7 +30,7 @@ Note: On OS X, if you have/had a "system" node installed and want to install mod
 
 Homebrew installation is not supported. If you have issues with homebrew-installed `nvm`, please `brew uninstall` it, and install it using the instructions below, before filing an issue.
 
-Note: If you're using `zsh` you can easily install `nvm` as a zsh plugin. Install [`zsh-nvm`](https://github.com/lukechilds/zsh-nvm) and run `nvm_update` to update.
+Note: If you're using `zsh` you can easily install `nvm` as a zsh plugin. Install [`zsh-nvm`](https://github.com/lukechilds/zsh-nvm) and run `nvm upgrade` to upgrade.
```

Comment:
I think just checking `nvm --help` for `nvm upgrade` might be easier and safer than checking the output of the command itself.